jwt/jwt.go

```diff
@@ -118,6 +118,44 @@ func getJWK(ctx context.Context, kid string) (*clerk.JSONWebKey, error) {
 	return nil, fmt.Errorf("no jwk key found for kid %s", kid)
 }
 
+// Returns the JSON web key set. Tries a cached value first, but if
+// there's no value or the entry has expired, it will fetch the set
+// from the API and cache the value.
+func getJWKSWithCache(ctx context.Context) (*clerk.JSONWebKeySet, error) {
+	cacheKey := "/v1/jwks"
```

Comment:
Hmmm.... I guess not until it's used some place else?

There's more that we can offer to avoid cache key collisions, but I think for now we're good.

Reply:
Mostly because it's not a "variable" if it's not changing. But yep, minor nit.

Reply:
You can always define it as a constant inside the function, it's a good middle ground I believe.

Reply:
Done!